project/Game.py

```diff
@@ -137,7 +153,15 @@ def quit_question(self, team, password):
         return True
 
     def answer_question(self, team, answer):
-        return True
+        if self.team.currentLandmark.answer is not answer:
```

Comment:
team.currentLandmark is an index number, it has no member answer. You have to go to game.landmarks[team.currentLandmark].answer